numpy/lib/function_base.py

```diff
@@ -2033,7 +2033,10 @@ def _vectorize_call(self, func, args):
             outputs = ufunc(*inputs)
 
             if ufunc.nout == 1:
-                res = array(outputs, copy=False, subok=True, dtype=otypes[0])
+                if iterable(outputs):
```

Comment:
Should this be `np.ndim(outputs) != 0` instead?

Reply:
It seemed to me that if the outputs were iterable, you would want `np.array` to touch it first before moving returning; that being said, I'm guessing `np.ndim` would work fine for checking as well. I've updated accordingly.